### ISSUE TITLE
Logic app

### ISSUE THREAD

**Issue by jinder79** [MicrosoftDocs/architecture-center#2337]
Where does Logic App fits in the compute services decision diagram? Please confirm. Thanks

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 510808e8-db5d-6b07-9a04-6edb743a2dfc
* Version Independent ID: 20d098aa-ef31-cb26-d677-765cfc9a7164
* Content: [Choosing an Azure compute service - Azure Architecture Center](https://docs.microsoft.com/en-us/azure/architecture/guide/technology-choices/compute-decision-tree)
* Content Source: [docs/guide/technology-choices/compute-decision-tree.md](https://github.com/microsoftdocs/architecture-center/blob/master/docs/guide/technology-choices/compute-decision-tree.md)
* Service: **architecture-center**
* Sub-service: **reference-architecture**
* GitHub Login: @doodlemania2
* Microsoft Alias: **pnp**

**Comment by absheik** [comment]
Thanks for reporting - this issue is under review.  This is a Microsoft Internal DevOps Bug ID AB#290642

**Comment by PramodValavala-MSFT** [comment]
@jinder79 Logic Apps is primarily a workflow orchestration SaaS service that can be used in many ways. For one, it could act as a serverless compute option for simpler business workflows instead of functions.

We have assigned this issue to the content author for further review.

**Comment by dsk-2015** [comment]
@jinder79 Thank you for your question. Logic Apps is a serverless workflow service, whereas Azure Functions is serverless compute. For this reason, the compute decision tree will not include Logic Apps. See this comparison for more information: https://docs.microsoft.com/en-us/azure/azure-functions/functions-compare-logic-apps-ms-flow-webjobs#compare-azure-functions-and-azure-logic-apps

I have added a note in this article for more clarification. Closing this issue.
